src/applications/disability-benefits/all-claims/components/IntroductionPage.jsx

```diff
@@ -120,9 +120,10 @@ class IntroductionPage extends React.Component {
               .
             </p>
           )}
-          <ol>
-            <li className="process-step list-one">
-              <h3 className="vads-u-font-size--h4">Prepare</h3>
+
+          <va-process-list uswds>
```

Comment:
Is `uswds` prop still needed? Default is `true` now

Reply:
Good catch. I think we can remove it for now. I just saw it in other usages in the codebase, but those are most likely older commits